src/commands/banCommands.ts

```diff
@@ -0,0 +1,147 @@
+import { ApplicationCommandOptionType, CommandInteraction } from "discord.js";
+import { Discord, Slash, SlashOption } from "discordx";
+import client, { getAuth } from "../main";
+import { Inject } from "typedi";
+import verifiedUserService from "../services/verifiedUserService";
+
+@Discord()
+export default class BanCommands {
+    @Inject()
+    verifiedUserService: verifiedUserService;
+    @Slash({ description: "Ban a user from community", name: "banlemmyuser" })
+    async banUser(
+        @SlashOption({

```

Comment:
Do like SlashOption(....)
argument:.....

dont "stack" those

Reply:
Corrected the SlashOption stacking, and moved the variables directly underneath the appropriate option